packages/generator/templates/app/src/app/page.tsx

```diff
@@ -0,0 +1,166 @@
+import Link from "next/link"
+import {invoke} from "./blitz-server"
+import {LogoutButton} from "./auth/components/LogoutButton"
```

Comment:
```suggestion
import {LogoutButton} from "./(auth)/components/LogoutButton"
```

The app does not render because the component was moved.

Reply:
Thanks for the report, will add it to list of things to fix. The issue here was that the imports were autochanged while I transferred fixes from a generated app.

Reply:
done